astroquery/esa/hubble/core.py

```diff
@@ -119,14 +119,79 @@ def download_product(self, observation_id, *, calibration_level=None,
 
         shutil.move(response, filename)
 
+    def get_member_observations(self, observation_id):
+        """
+        Parameters
+        ----------
+        observation_id : str, mandatory
+            Observation identifier.
+
+        Returns
+        -------
+        A list of strings with the observation_id of the associated
+        observations that can be used in get_product_list and
+        get_obs_products functions
+        """
+        if observation_id is None:
+            raise ValueError(self.REQUESTED_OBSERVATION_ID)
+        observation_type = self.get_observation_type(observation_id)
+
+        if 'Composite' in observation_type:
+            oids = self._select_members(observation_id)
+        elif 'Simple' in observation_type:
+            oids = self._select_composite(observation_id)
+        else:
+            raise ValueError("Invalid observation id")
+        return oids
+
+    def get_hap_hst_link(self, observation_id):
+        if observation_id is None:
+            raise ValueError(self.REQUESTED_OBSERVATION_ID)
+        observation_type = self.get_observation_type(observation_id)
+        if 'Composite' in observation_type:
+            raise ValueError("HAP-HST link is only available for simple observations. Input observation is Composite.")
+        elif 'HAP' in observation_type:
+            oids = self._select_members(observation_id)
+        elif 'HST' in observation_type:
+            query = f"select observation_id from ehst.observation where obs_type='HAP Simple' and members like '%{observation_id}%'"
+            job = self.query_hst_tap(query=query)
+            oids = job["observation_id"].pformat(show_name=False)
+        else:
+            raise ValueError("Invalid observation id")
+        return oids
```

Comment:
In my opinion these two methods should not be merged as there is a big difference. The get_hap_hst_link method assumes that you only want the link between a simple hst observation and a simple hap observation. If we were to merge the two methods together the method would have to be changed so that it can distinguish whether or not the user wants the hap to hst link or they want to find the composite observations made up by this simple observation. This change would require the user to input extra variables and the method will start to get complicated. I personally feel two simple methods will be better for the user than one more complicated one. 

I am not sure if I have explained myself very well but I am happy to discuss this further. If you still feel it is better to merge these methods than I am happy to do so.

Reply:
this sounds good, thanks for the explanation.